client/landing/stepper/declarative-flow/internals/steps-repository/launchpad/launchpad-site-preview.tsx

```diff
@@ -33,8 +33,12 @@ const LaunchpadSitePreview = ( { siteSlug }: { siteSlug: string | null } ) => {
 		} );
 	}
 
+	function preventTabbingToIFrame(): void {
+		( document.querySelector( 'iframe.web-preview__frame' ) as HTMLIFrameElement ).tabIndex = -1;
```

Comment:
A couple thoughts here:
1. I want to be cautious about relying on delivering functionality via css selectors. We have less control / higher probability that the functionality could break as a result of something being changed or refactored, and more difficult for folks making those changes to know we are relying on these items.
2. That said, I wouldn't necessarily expect these selectors to change... but you never know 🤷‍♀️ 

So other ideas could be to update the `WebPreview` component itself to either:
* Always supply this -1 tab index... if it seems like that makes sense for all cases?
* Or maybe a more safe change being to add a prop to the WebPreview component whose existence conditionally  adds this tabIndex to that element.